ownCloudAppShared/Client/Collection Views/Cells/MessageCell.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  MessageCell.swift
+//  ownCloudAppShared
+//
+//  Created by Felix Schwarz on 27.05.22.
+//  Copyright © 2022 ownCloud GmbH. All rights reserved.
+//
+
+/*
+ * Copyright (C) 2022, ownCloud GmbH.
+ *
+ * This code is covered by the GNU Public License Version 3.
+ *
+ * For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+ * You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+ *
+ */
+
+import UIKit
+import ownCloudSDK
+import ownCloudApp
+
+class MessageCell : ThemeableCollectionViewListCell {
```

Comment:
Does not implement code. Is it currently needed?

Reply:
Yes, that was a leftover from an idea I didn't yet get around to execute. I removed it now.